modules/ROOT/pages/user-management.adoc

```diff
@@ -24,43 +24,187 @@ You can access the **Settings** page by selecting **Settings** from the sidebar
 
 Each project can have multiple users with individual accounts allowing access to the same environment.
 
-The users with access to a project can be viewed and managed from the **Users** page. 
+The users with access to a project can be viewed and managed from the **Users** page.
 You can access the **Users** page by selecting **Users** from the sidebar menu of the console.
 
-=== Roles
+[[roles]]
+== Roles
 
-Users within a project can be assigned one of the following roles:
+Users within a project can be assigned one of the following predefined roles:
 
 * _Project Admin_
 * _Project Member_
 * _Project Viewer_
 * _Project Metrics Integration Reader_
 
+These roles grant the users certain privileges both on the console level as well as on the instance level.
+The roles are immutable and every new user needs to be assigned one.
+
 :check-mark: icon:check[]
 
-.Roles
+.Roles and console capabilities
 [opts="header",cols="3,1,1,1"]
 |===
-| Capability | Admin | Member | Viewer
+| Capability | Viewer | Member | Admin
 | View users and their roles | {check-mark} | {check-mark} | {check-mark}
 | View and open instances | {check-mark} | {check-mark} | {check-mark}
-| Access the Neo4j Customer Support Portal | {check-mark} | {check-mark} | {check-mark} 
-| Perform all actions on instances footnote:[Actions include creating, deleting, pausing, resuming, and editing instances.] | {check-mark} | {check-mark} |
-| Clone data to new and existing instances | {check-mark} | {check-mark} |
-| Take on-demand snapshots | {check-mark} | {check-mark} |
-| Restore from snapshots | {check-mark} | {check-mark} |
-| Edit the project name | {check-mark} | |
-| Invite new users to the project | {check-mark} | |
-| Edit existing users' roles | {check-mark} | |
-| Delete existing users from the project | {check-mark} | |
-| View and edit billing information | {check-mark} | |
+| Access the Neo4j Customer Support Portal | {check-mark} | {check-mark} | {check-mark}
+| Perform all actions on instances footnote:[Actions include creating, deleting, pausing, resuming, and editing instances.] | | {check-mark} | {check-mark}
+| Clone data to new and existing instances | | {check-mark} | {check-mark}
+| Take on-demand snapshots | | {check-mark} | {check-mark}
+| Restore from snapshots | | {check-mark} | {check-mark}
+| Edit the project name | | | {check-mark}
+| Invite new users to the project | | | {check-mark}
+| Edit existing users' roles | | | {check-mark}
+| Delete existing users from the project | | | {check-mark}
+| View and edit billing information | | | {check-mark}
 |===
 
 [NOTE]
 ====
 Each project must have at least one Project Admin, but it is also possible for projects to have multiple Project Admins.
 ====
 
+Additionally, predefined roles are assigned certain privileges on the instance level as well.
+
+.Roles and database privileges
+[options="header", cols="3,1,1,1,1,1"]
+|===
+| Privilege
+| Viewer
+| Member
+| Admin Free
+| Admin Pro
+| Admin BC
```

Comment:
Is `Admin BC` widely used in this part of the docs? Or do you think it would be better to expand it?

Reply:
Good point, I put that there mainly for brevity, but should probably write that out, space permitting.